pkgs/tools/networking/bandwhich/default.nix

```diff
@@ -2,13 +2,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "bandwhich";
-  version = "unstable-2023-09-11";
+  version = "0.21.0";
 
   src = fetchFromGitHub {
     owner = "imsnif";
     repo = pname;
-    rev = "eba356220cc06254b96cd3241bc80ab7a0ab017b";
-    hash = "sha256-qrVGUbVbURXSKJy28mhpwVkzSGgct8PPOnwb0FdVWtE=";
+    rev = if version == "0.21.0" then "v${version}" else "eba356220cc06254b96cd3241bc80ab7a0ab017b";
```

Comment:
Why the if?

Reply:
oops that was a hack I used for nix-update to generate the correct commit message, it should be good now